### DISCUSSION TITLE
包

### DISCUSSION THREAD

**Discussion by salvaters** [npm/feedback#657] [Registry] [answered]
vue-ele-formcoms，不小心废弃了这个包能帮我解除废弃么，如果解除不了能帮我将该包删除了么

**Comment by monishcm** [comment]
Please open a support ticket.

**Reply by salvaters** [reply]
> Please open a support ticket.
我提交了邮件！

**Comment by MylesBorins** [comment, marked as answer]
Once your download count goes under 300 you'll be able to delete the package, in the mean time you will unfortunately need to wait. Newly published packages often see 300 - 1000 downloads from bots when being published, so in 1 - 2 weeks the download count should get low enough to allow you to delete it.
